spec/factories/legal_aid_applications.rb

```diff
@@ -145,5 +149,48 @@
       transaction_period_start_at { 3.months.ago.beginning_of_day }
       transaction_period_finish_at { Time.now.beginning_of_day }
     end
+
+    trait :test_application_initiated do
```

Comment:
The `test_application_` prefix is too verbose and assumes these traits will only be useful in the current context. I'd suggest replacing it with `at_` so: `at_initiated`, `at_checking_answers`. In tests we can then do:
```ruby
let(:legal_aid_application) { create :legal_aid_application, :at_checking_passported_answers }
```
I'd also check to see if any duplication can be removed. For example, I expect specs currently using the trait `:mean_completed` could use the new `at_means_completed` and therefore the spec could be updated and the now redundant trait removed.

Reply:
I suggested Abdi to use traits with `test_application_ `, but `at_` is much better and concise 👍